tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -53,7 +53,8 @@ else
         fi
         docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} -f $DOCKER_FILE .
     elif [ $BUILD_DEVICE = "tensorrt" ]; then
-        IMAGE="ubuntu16.04-cuda10.0-cudnn7.4-tensorrt5.0"
+        # TensorRT container release 19.06
+        IMAGE="ubuntu16.04-cuda10.1-cudnn7.6-tensorrt5.1"
```

Comment:
use $BUILD_OS and $CUDA_VER instead of hardcoding ubuntu16.04 and cuda10.1-cudnn7.6 etc.

why support ubuntu16/tensorrt5.1 ? your EP changes are no longer compatible with tensorrt 5.1 ?

Reply:
Current TRT EP changes have TRT6 specific APIs and it isn't compatible with TRT5. Do we need to make the EP backward compatible?

Reply:
right. that's why i made the comment on why you still have ubuntu16 trt 5.1 here.
let's remove it and explicitly document that you need to use trt 6 with ORT moving forward.
we don't want to maintain both trt 5 and trt 6 api's.